api/hypershift/v1alpha1/hostedcluster_types.go

```diff
@@ -2019,6 +2019,12 @@ type HostedClusterStatus struct {
 	// +patchStrategy=merge
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 
+	// Payload represents the CPU architecture type of the HostedCluster.Spec.Release.Image. The valid values are:
+	// Multi, ARM64, AMD64, or PPC64LE.
+	// +kubebuilder:validation:Enum=Multi;ARM64;AMD64;PPC64LE
+	// +optional
+	Payload string `json:"payload,omitempty"`
```

Comment:
Would it make sense to use another name that represents that this is the arch? `PayloadArch` or `ReleaseArch` maybe?

Reply:
Why are we adding the field in the `v1alhpa1` if we already have a `v1beta1` API ?

Reply:
+1

Reply:
Updated